Targets/AppStoreConnectUI/Sources/Authorization/AuthView.swift

```diff
@@ -6,27 +6,49 @@
 //
 
 import SwiftUI
+import UniformTypeIdentifiers
 
-struct AuthView: View {
-    @StateObject var viewModel: AuthViewModel
+public struct AuthView: View {
+    @StateObject private var viewModel: AuthViewModel
+    @State private var isImporting: Bool = false
     
-    var body: some View {
+    public init(viewModel: AuthViewModel) {
+        self._viewModel = StateObject(wrappedValue: viewModel)
```

Comment:
Better not to use, but t is okay right now )

```suggestion
        self._viewModel = StateObject(wrappedValue: viewModel)
```

Reply:
Please clarify how to awoid this and still keep ability to have this initializer with viewModel for swiftUI view?

Reply:
Here is a good article about this, you can also look at the default autogenerated init for state object. Let's do the same, auto closure with viewModel.

https://medium.com/@mega-blog/shall-we-make-all-stateobject-properties-private-in-swiftui-a844c7b576c9